packages/suite/src/components/suite/Settings/components/DeviceBanner.tsx

```diff
@@ -35,11 +46,21 @@ interface DeviceBannerProps {
 
 export const DeviceBanner = ({ title, description }: DeviceBannerProps) => {
     const { device } = useDevice();
+
+    const { transport } = useSelector(state => ({
+        transport: state.suite.transport,
+    }));
+
+    const showWebUsb = isWebUSB(transport);
```

Comment:
All case styles are used for usb/Usb/USB in this file. What about unifying it to USB. It's a shortcut, so every letter stands for one word...

Reply:
Also `showWebUsb` is a little confusing to me, maybe `isWebUSBTransport` or not name it at all and use it directly in the condition, or create a const for the whole condition such a `const showWebUSBButton = !device?.connected && isWebUSB(transport);`?

Reply:
`USB` is not good for camelCase? `isWebUSBTransport` looks weird. I am for `isWebUsbTransport`

Reply:
`isWebUniversalServiceBusTransport` looks good, but sounds weird 😉  

 `isWebUSBTransport` doesn't look weird to me, it's consistent. But no matter what, would be nice to have it consistent.

Reply:
b09d9daf4226a80fb9f52b73e4d9d9fb2362ead1 + a new commit containing some refactoring